reference/upgrade/README.md

```diff
@@ -72,6 +72,8 @@ As we are aiming to always provide you more control and flexibility on how to de
 
 The `web` key is now a set of `locations` where you can define very precisely the behavior of each URL prefix.
 
+Note, we no longer move your application from "/" to "public/" automatically if the new format is adopted.
```

Comment:
I was actually the one to ask this, because we were surprised that this "magically" changed (technically, it magically changed before and now it no longer does).

Adding a suggestion for a fix might be helpful, because it actually doesn't seem completely obvious, sometehing as simple as mv * public as build step for example doesn't work as public/sites is auto-created. Still working out the best approach for moving it.